src/udiskslinuxmanager.c

```diff
@@ -1013,6 +1013,62 @@ handle_can_repair (UDisksManager         *object,
   return TRUE;
 }
 
+
+static GList*
+get_block_objects (UDisksManager *manager,
+                   guint         *num_blocks)
+{
+  UDisksLinuxManager *linux_manager = UDISKS_LINUX_MANAGER (manager);
+  GDBusObjectManagerServer *object_manager = NULL;
+  GList *objects, *objects_p = NULL;
+  GList *ret = NULL;
```

Comment:
This could use ``GSList`` which would save some memory, but it probably just doesn't matter here.